Makefile

```
@@ -35,10 +37,19 @@ ignition-validate-cross:
 
 .PHONY: install
 install:
-	for x in dracut/*; do \
-	  bn=$$(basename $$x); \
-	  install -m 0644 -D -t $(DESTDIR)/usr/lib/dracut/modules.d/$${bn} $$x/*; \
-	done
+install: all
```

checkmake (0.3.2)

[warning] Target body for "install" exceeds allowed length of 5 lines (12).

(maxbodylength)